downloads/0.9.x-release-notes/0.970.0/RELEASE_NOTE.html

```diff
@@ -424,8 +424,8 @@ <h2>Messaging endpoints</h2>
 }
 </code></pre>
 
-<h1>Standard Library</h1>
-<p>The Ballerina standard library provides a set of commonly used functionalities. The following packages are available as a part of the standard library:</p>
+<h1>Ballerina library</h1>
```

Comment:
Avoid release notes, blogs, etc

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.